cli/command/stack/kubernetes/cli.go

```diff
@@ -46,7 +46,7 @@ func AddNamespaceFlag(flags *flag.FlagSet) {
 }
 
 // WrapCli wraps command.Cli with kubernetes specifics
-func WrapCli(dockerCli command.Cli, opts Options) (*KubeCli, error) {
+func WrapCli(dockerCli command.Cli, opts Options, orchestrator command.Orchestrator) (*KubeCli, error) {
```

Comment:
Can't `orchestrator` be put inside `Options` somehow?

Reply:
I'll try that!